app/code/Magento/CatalogGraphQl/etc/graphql/di.xml

```diff
@@ -80,4 +80,26 @@
     </virtualType>
     <preference for="Magento\Framework\Search\Adapter\Mysql\Query\Builder\Match"
                 type="Magento\CatalogGraphQl\Model\Search\Adapter\Mysql\Query\Builder\Match" />
+    <type name="Magento\StoreGraphQl\Model\Resolver\Store\StoreConfigDataProvider">
+        <arguments>
+            <argument name="extendedConfigData" xsi:type="array">
```

Comment:
@VitaliyBoyko Please, check(grep) that all of the items are used and needed on frontend
For example, I have a debt about `flat_catalog_category`...

Reply:
@zetlen Could you please take a look for more configs which are might be redundant. Thank you!

Reply:
@VitaliyBoyko just grep all of the options in code
If option is not used on frontend (and frontend logic is not based on option) need to delete it from list

Reply:
I think you're right that we don't need `flat_catalog_category`. Let me list what I can see now will be useful to the PWA frontend in the future. **Anything I _don't_ list here can be removed from this PR if you need to.**

### SEO
- `product_url_suffix`: PWA **definitely** needs this to do accurate SEO routing; these configurations tell our router how to derive the URL key from the URL pathname.
- `category_url_suffix`: PWA **definitely** needs this, see above
- `product_use_categories`: PWA **definitely** needs this, see above
- `title_separator`: PWA can use this to render dynamic page titles.

### Display
- `list_mode`: PWA can use this to control category/search rendering and pagination.
- `grid_per_page_values`: See above
- `list_per_page_values`: See above
- `grid_per_page`: See above
- `list_per_page`: See above
- `catalog_default_sort_by`: See above

Nothing else from the Catalog config section is obviously useful to us (yet). @jimbo and @soumya-ashok and @tjwiebell and @supernova-at might have a different opinion based on UX best practice or known UI gaps, but I think this is plenty.

Thanks again!